components/brave_wallet/browser/eth_tx_controller.h

```diff
@@ -98,22 +98,24 @@ class EthTxController : public KeyedService,
       const std::string& tx_meta_id,
       const std::vector<uint8_t>& data,
       SetDataForUnapprovedTransactionCallback callback) override;
-  void ApproveHardwareTransaction(
+  void GetNonceForHardwareTransaction(
       const std::string& tx_meta_id,
-      ApproveHardwareTransactionCallback callback) override;
-  void ProcessLedgerSignature(const std::string& tx_meta_id,
-                              const std::string& v,
-                              const std::string& r,
-                              const std::string& s,
-                              ProcessLedgerSignatureCallback callback) override;
-
+      GetNonceForHardwareTransactionCallback callback) override;
   void SpeedupOrCancelTransaction(
       const std::string& tx_meta_id,
       bool cancel,
       SpeedupOrCancelTransactionCallback callback) override;
   void RetryTransaction(const std::string& tx_meta_id,
                         RetryTransactionCallback callback) override;
-
+  void ProcessHardwareSignature(
+      const std::string& tx_meta_id,
+      const std::string& v,
```

Comment:
can we give these more descriptive names than v, r and s?

Reply:
I guess this also goes deep into the code so probably needs a separate PR to address

Reply:
this should be v,r and s, specific things for Ethereum transactions https://github.com/trezor/connect/blob/develop/src/js/types/networks/ethereum.js#L65